src/main/scala/ai/starlake/tests/StarlakeTests.scala

```diff
@@ -788,6 +804,23 @@ object StarlakeTestData {
   ): Option[StarlakeTest] = {
     val taskOrTableFolderName = testFolder.getParentFile.getName
     val domainName = testFolder.getParentFile.getParentFile.getName
+
+    // All files that do not start with an '_' and end with .sql are considered pretest sql statements
+    val preTestStatementsFiles = testFolder
+      .listFiles(f => f.isFile && !f.getName.startsWith("_") && f.getName.endsWith(".sql"))
+      .toList
+    val preTestStatements: List[String] =
+      preTestStatementsFiles.flatMap { preTestStatementsFile =>
+        if (preTestStatementsFile.exists()) {
+          val source = Source.fromFile(preTestStatementsFile)
+          val sql = source.mkString
+          source.close()
+          sql.split(";")
```

Comment:
I am not sure that you need to split on ';' and the way it is done is not very safe.
I think that you can just submit all of them